build/nl_server/Dockerfile

```diff
@@ -26,7 +26,7 @@ COPY nl_server/requirements.txt /workspace/nl_server/requirements.txt
 COPY nl_requirements.txt /workspace/nl_requirements.txt
 # --no-cache-dir removes ~/.cache files, which can be a few GB.
 RUN pip install --upgrade pip setuptools
-RUN pip3 install torch==2.2.2 --extra-index-url https://download.pytorch.org/whl/cpu
+RUN pip3 install torch==$(grep '^torch==' /workspace/nl_requirements.txt | cut -d'=' -f3) --extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
since torch is already included in `nl_requirements.txt`, do we still need this line? 

can we make the entry in the nl_requirements.txt like:

```
torch==2.8.0 --find-links https://download.pytorch.org/whl/cpu
```